### DISCUSSION TITLE
Allow clearing cache in extensions through the UtilsService

### DISCUSSION THREAD

**Discussion by nodeworks** [directus/directus#19945] [Feature Requests]
### Summary

Right now you can't clear cache in extensions easily. You would have to make a POST request to the endpoint with an admin token. There should be a method on the UtilsService that allows to clear cache.

### Motivation

Need to clear cache in extensions.

### Detailed Design

Create a new method on the UtilsService to allow clearing cache in extensions.

### Should Have:
- A new method on the UtilsService for clearing cache.

### Drawbacks

No drawbacks?

### Alternatives

None

### Adoption Strategy

There are no breaking changes, just an additional method.

**Comment by rijkvanzanten** [comment]
I don't see why not. Wanna open a PR for that @nodeworks? Happy to just accept this as a small improvement rather than a new feature 👍🏻

**Comment by rijkvanzanten** [comment]
This feature request has been accepted and is queued to be implemented! You can follow along with the progress here: https://github.com/directus/directus/issues/19958
